File: Android/app/src/main/java/com/khsm/app/data/entities/Meeting.java
package com.khsm.app.data.entities;

import java.io.Serializable;
import java.util.Date;

public class Meeting implements Serializable {
    public final Integer id;
    public final Integer number;
    public final Date date;

    public Meeting(Integer id, Integer number, Date date){
        this.id = id;
        this.number = number;
        this.date = date;
    }
}

File: Backend/Data/Entities/Meeting.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Backend.Data.Entities
{
    public class Meeting
    {
        [Required]
        public int Id { get; set; }
        [Required]
        public int Number { get; set; }
        [Required]
        public DateTime Date { get; set; }

        protected bool Equals(Meeting other)
        {
            return Id == other.Id;
        }

        public override bool Equals(object obj)
        {
            if (ReferenceEquals(null, obj)) return false;
            if (ReferenceEquals(this, obj)) return true;
            if (obj.GetType() != this.GetType()) return false;
            return Equals((Meeting) obj);
        }

        public override int GetHashCode()
        {
            return Id;
        }
    }
}